TicketDesk/ngWebClient-StandAlone/src/app/ticket-center/ticket-center.component.html

```diff
@@ -1,5 +1,7 @@
-<ngb-tabset justify="fill" (tabChange)="onTabChange">
-  <ngb-tab *ngFor="let tabName of tabNames" [class.listName]="tabName" title="{{tabName}}">
+<div class="container-fluid">
+<div class="card">
+<ngb-tabset justify="fill" (tabChange)="onTabChange($event)">
+  <ngb-tab *ngFor="let tabName of tabNames" id="{{tabName}}" title="{{convertListNameToDisplayStr(tabName)}}">
```

Comment:
.........okay, but shouldn't there still be {{ tabName }} spaces even if in string